tools/server_settings_replicator/server_settings_replicator.rb

```diff
@@ -0,0 +1,31 @@
+class ServerSettingsReplicator
+  def self.replicate(server, path_string, dry_run = false)
+    path = path_string.split("/").map(&:to_sym)
+
+    # all servers except source
+    target_servers = MiqServer.where.not(:id => server.id)
+    settings = construct_setting_tree(path, server.settings_for_resource.fetch_path(path).to_h)
+
+    puts "Replicating from server id=#{server.id}, path=#{path_string} to #{target_servers.count} servers"
+    puts "Settings: #{settings}"
```

Comment:
![](http://screenshots.chrisarcand.com/g8gc7.jpg)

NoooOooooOoooooooo

Reply:
![giphy](https://user-images.githubusercontent.com/19339/30715389-27cd5d32-9ee4-11e7-9f58-723ffd533ed7.gif)

Reply:
good catch! This was started as a simple tool script that no spec was intended and it outgrew...
I have another PR to silent them coming.

Reply:
The PR is https://github.com/ManageIQ/manageiq/pull/16023